go/stellar/build.go

```diff
@@ -18,6 +19,107 @@ import (
 	stellarAddress "github.com/stellar/go/address"
 )
 
+func ShowAdvancedSendBanner(mctx libkb.MetaContext, remoter remote.Remoter, from stellar1.AccountID, to string) (shouldShow bool, err error) {
```

Comment:
Maybe `ShouldShowAdvancedSendBanner` or `ShouldOfferAdvancedSend`. `ShowAdvancedSendBanner` sounds like a function that does the showing